docs/content/en/docs/community/docs_contributing.md

```diff
@@ -43,9 +43,16 @@ EKS Anywhere documentation uses the [Hugo](https://gohugo.io/categories/fundamen
 
 * **General style issues**: Unless otherwise instructed, follow the [Kubernetes Documentation Style Guide](https://kubernetes.io/docs/contribute/style/style-guide/) for formatting and presentation guidance.
 
+* **Creating images**: Screen shots and other images should be published in PNG format.
```

Comment:
Do we care if they are exported with a background transparency or not? How about image sizing? Is there a recommended size we want to suggest?

Reply:
I use a white background. Hadn't thought about transparency, but I don't see why we would want that. I would generally discourage image resizing. It can be done with html markup, but I think we want to keep it pretty simple. Hugo/docsy is not really meant for fancy layouts. In terms of image size, the most important thing it to crop the images in a way that focuses on the content being highlighted. In other words, don't show a whole web page screenshot if you are just focusing on one field.

Reply:
I added a few words along these lines to the content below.